apps/showcase/src/app/app.module.ts

```diff
@@ -108,7 +109,8 @@ export function registerCustomComponents(): Map<string, any> {
     C11nModule.forRoot({registerCompFunc: registerCustomComponents}),
     StylingDevtoolsModule,
     LocalizationDevtoolsModule,
-    ConfigurationDevtoolsModule
+    ConfigurationDevtoolsModule,
```

Comment:
Not for your PR but shouldn't we move to a standalone application ?

Reply:
One day

Reply:
I open the issue so we will do it one day :D
https://github.com/AmadeusITGroup/otter/issues/2233